tests/components/test_auto_fill.py

```diff
@@ -95,6 +98,47 @@ def test_auto_fill_validation_missing_entities(kwik_e_mart_app, qa_kwik_e_mart):
     )
 
 
+def handler_sub(request, responder):
```

Comment:
To verify that ```handler_sub``` is invoked, you can actually pass in a mock function and assert that the mock function is called.

```
In [11]: from mock import MagicMock

In [12]: f = MagicMock()

In [15]: f.assert_called_once_with(1, 2)
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-15-4231415d7976> in <module>
----> 1 f.assert_called_once_with(1, 2)

~/.pyenv/versions/3.6.8/envs/mm/lib/python3.6/site-packages/mock/mock.py in assert_called_once_with(_mock_self, *args, **kwargs)
    945             msg = ("Expected '%s' to be called once. Called %s times." %
    946                    (self._mock_name or 'mock', self.call_count))
--> 947             raise AssertionError(msg)
    948         return self.assert_called_with(*args, **kwargs)
    949

AssertionError: Expected 'mock' to be called once. Called 0 times.

In [16]: f(1, 2)
Out[16]: <MagicMock name='mock()' id='4619359232'>

In [17]: In [15]: f.assert_called_once_with(1, 2)

Reply:
Awesome, added.